main_test.go

```diff
@@ -53,6 +53,61 @@ func TestSendMetrics(t *testing.T) {
 	assert.NoError(err)
 }
 
+func TestSendMetricsHostStripping(t *testing.T) {
+	var hostStrippingDataSet = []struct {
+		entityName string
+		expectedBody string
+		stripHost    bool
+	}{
+		{
+			entityName: "sensu.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=sensu.company.com value=42`,
+			stripHost: true,
+		},
+		{
+			entityName: "answer.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=answer.company.com value=42`,
+			stripHost: true,
+		},
+		{
+			entityName: "prod-eu-db01.company.com",
+			expectedBody: `answer,foo=bar,sensu_entity_name=prod-eu-db01.company.com value=42`,
+			stripHost: true,
+		},
+		{
+			entityName: "no-stripping.company.com",
+			expectedBody: `no-stripping,foo=bar,sensu_entity_name=no-stripping.company.com company.com.answer=42`,
+			stripHost: false,
+		},
+		{
+			entityName: "company.com",
+			expectedBody: `company,foo=bar,sensu_entity_name=company.com com.answer=42`,
+			stripHost: false,
+		},
```

Comment:
`company.com.answer=42` and `com.answer=42` don't look right, but since `stripHost: false`, it has nothing to do with your changes.. just very peculiar.

Reply:
Related to this: https://github.com/sensu/sensu-go/blob/master/api/core/v2/metrics.go#L24